cmd/skopeo/inspect.go

```diff
@@ -97,7 +99,13 @@ var inspectCmd = cli.Command{
 			outputData.Name = dockerImg.SourceRefFullName()
 			outputData.RepoTags, err = dockerImg.GetRepositoryTags()
 			if err != nil {
-				return fmt.Errorf("Error determining repository tags: %v", err)
+				// some registries may decide to block the "list all tags" endpoint
+				// gracefully allow the inspect to continue in this case
+				if !strings.Contains(err.Error(), "401") {
```

Comment:
[BTW it would be nice to have a documented example of “some registries”, perhaps naming a product or a product version or a config option, either in the code comment or at least mentioned in this PR.

That would allow other users to find this PR and to realize that they need to upgrade.